src/test/java/botto/util/ParserTest.java

```diff
@@ -0,0 +1,54 @@
+package botto.util;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import botto.BottoException;
+import botto.command.AddDeadlineCommand;
+import botto.command.AddEventCommand;
+import botto.command.AddToDoCommand;
+import botto.command.DeleteCommand;
+import botto.command.ExitCommand;
+import botto.command.MarkDoneCommand;
+import botto.command.ShowListCommand;
+
+public class ParserTest {
+
+    @Test
+    public void parse_allCommands_success() throws BottoException {
```

Comment:
Although it is more tedious, it might be better to split each command type into its own test function. But that's not a big deal.